frontend/src/pages/Contracts/Contract/InvoiceProcessing/InvoiceDeliverables/fields.tsx

```diff
@@ -0,0 +1,38 @@
+import { FormikValues } from "formik";
+import { UseQueryResult } from "react-query";
+import { IEditFields } from "types";
+
+/**
+ * The view fields.
+ *
+ * @param   {UseQueryResult<FormikValues>} query The react query data for specific project.
+ * @returns {Array}
+ */
+export const readFields = (query: UseQueryResult<FormikValues>) => {
+  return [
+    { width: "half", title: "Deliverables / Expense", value: query?.data?.resource_id },
+    //{ width: "half", title: "Type", value: query?.data?.hours },
```

Comment:
Is this left in intentionally?

Reply:
Sort of. The mockup shows a Type field but I'm not sure where in the database it comes from. The resource and deliverable read/edit forms are just kind of placeholders until their actual implementation is figured out in the next subtask.